packages/manager/apps/pci-instances/src/pages/instances/instance/dashboard/components/InstanceNetworkingBlock.component.tsx

```diff
@@ -102,20 +95,22 @@ const InstanceNetworkingBlock: FC = () => {
     [projectUrl, t, instance?.isEditEnabled],
   );
 
+  const expandedNetworks = instance?.privateNetwork.slice(2) ?? null;
```

Comment:
```suggestion
  const expandedNetworks = instance?.privateNetwork.slice(2);
```

Empty array and undefined do the same with '&&' line 130. No nedd to add null